src/builder/edit_channel.rs

```diff
@@ -287,6 +289,14 @@ impl<'a> EditChannel<'a> {
         self.default_forum_layout = Some(default_forum_layout);
         self
     }
+    /// The status of a voice channel.
+    /// The user must be within the channel to change the status but if the
+    /// user has Manage Channels the user can reset it without being in the channel.
+    /// Can be None, an empty string or up to 500 characters.
+    pub fn status(mut self, status: impl Into<Option<String>>) -> Self {
+        self.status = status.into();
+        self
+    }
```

Comment:
Shouldn't this be done for the special [Set Voice Channel Status](https://github.com/discord/discord-api-docs/pull/6400/files) endpoint instead of the Edit Channel endpoint?

Reply:
I guess you are probably right, yeah setting it to None *does* work (haven't tested trying to set it to an actual status because I haven't set up joining vcs on my bot yet) but it fires CHANNEL_UPDATE instead of VOICE_CHANNEL_STATUS_UPDATE so yeah I'm guessing this needs to be moved.

Any pointers on how I could implement the set voice channel status endpoint?

Reply:
I have figured out how to do this but it currently has an experiment lock so it won't actually do anything yet. I think its best if we remove the changing status code from this pull request and split it off into a different pull request until it no longer has this limitation.